filebeat/docs/multiline.asciidoc

```diff
@@ -220,6 +220,21 @@ multiline.flush_pattern: 'End event'
 
 The `flush_pattern` option, specifies a regex at which the current multiline will be flushed. If you think of the `pattern` option specifying the beginning of an event, the `flush_pattern` option will specify the end or last line of the event.
 
+Please note this example will not work correctly if start/end log blocks are mixed with non-multiline logs, or if different start/end log blocks overlap with each other. For instance, `Some other log` log lines in the example below will be merged into a _single_ multiline document because they neither match `multiline.pattern` not `multiline.flush_pattern` and `multiline.negate` is set to `true`.
```

Comment:
```suggestion
NOTE: This example will not work correctly if start/end log blocks are mixed with non-multiline logs, or if different start/end log blocks overlap with each other. For instance, `Some other log` log lines in the following example will be merged into a _single_ multiline document because they neither match `multiline.pattern` not `multiline.flush_pattern`, and `multiline.negate` is set to `true`.
```
```suggestion
Please note this example will not work correctly if start/end log blocks are mixed with non-multiline logs, or if different start/end log blocks overlap with each other. For instance, `Some other log` log lines in the example below will be merged into a _single_ multiline document because they neither match `multiline.pattern` not `multiline.flush_pattern` and `multiline.negate` is set to `true`.
```